### ISSUE TITLE
Please support https

### ISSUE THREAD

**Issue by thommyhh** [pangolinjs/core#38]
Current scenario:
* Application uses users location (which is only allowed from secure location, which includes localhost)
* Dev server is run through `træfik` reverse proxy with a different host name as `localhost` so I need to use HTTPS.
* The websocket url is always `ws://`, but it should be `wss://` if the page is served via HTTPS. 
* To support `ws://` and `wss://` at the same time, we need a different port for `wss://` (like http and https uses different ports). `træfik` would handle this, so the websocker server might still only listen on port 3001,  but the port in the browser needs to be different.

**Comment by thommyhh** [comment]
@mvsde You probably missed the last point in the feature request. WS and WSS should be possible at the same time, so we need two different ports in the frontend, but only one port the websocket server is listening on, as both ports are proxied to the same port by træfik.

Would be nice, if you can accomplish this.

**Comment by mvsde** [comment]
I think I found a great solution for this problem:

The `webpack-dev-server` allows [URL proxying](https://webpack.js.org/configuration/dev-server#devserverproxy) with support for WebSockets. Instead of connecting directly to the socket from the UI, I configured a proxy URL (https://pangolinjs.org/guide/configuration.html#dev-server). So both the HTTP dev server and the WebSocket dev server will be available through the same host and port.
